lib/Locale/Country/Extra.pm

```diff
@@ -33,7 +33,14 @@ sub code_from_country {
     my ($self, $country) = @_;
 
     my %code_countries = reverse %{ $self->_country_codes };
-    return lc $code_countries{$country};
+
+    unless( exists $code_countries{$country} ){
+        $country = lc $country;
+        $country = List::Util::first { $country eq lc $_ } keys %code_countries;
```

Comment:
We'll still need a `use List::Util ();` declaration above at the top.

Reply:
sorry, I missunderstanded it, update it again